src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -45,7 +46,9 @@
            path (core/get-master-mount ::master-service route-config)
            ring-handler (when path
                           (core/get-wrapped-handler
-                            (-> (core/root-routes handle-request jruby-service)
+                            (-> (core/root-routes handle-request
+                                                  jruby-service
+                                                  get-code-content)
```

Comment:
@haus this is what we were expecting to happen, the last time we were working on this code, right?